src/main/java/core/basesyntax/model/RightTriangle.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.model;
+
+import core.basesyntax.model.abstraction.Color;
+
+public class RightTriangle extends Rectangle {
```

Comment:
```suggestion
public class RightTriangle extends Figure {
```
You can't do that, because they are completely different figures